app/views/users/index.html.erb

```diff
@@ -1,29 +1,37 @@
+<% provide :title, "All user" %>
 <p id="notice"><%= notice %></p>
+<div class="container">
+  <div class="row">
+    <div class="box">
+      <div class="col-md-6 col-md-offset-3 login">
+        <hr>
+        <h1 class="intro-text text-center">
+            <strong>All users</strong>
+        </h1>
+        <hr>
+        <table>
+          <thead>
+            <tr>
+            <th>Name</th>
+            <th>Email</th>
+            <th colspan="3"></th>
+            </tr>
+          </thead>
 
-<h1><%= Hello %></h1>
-
-<table>
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Email</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @users.each do |user| %>
-      <tr>
-        <td><%= user.name %></td>
-        <td><%= user.email %></td>
-        <td><%= link_to 'Show', user %></td>
-        <td><%= link_to 'Edit', edit_user_path(user) %></td>
-        <td><%= link_to 'Destroy', user, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
-
-<br>
-
-<%= link_to 'New User', new_user_path %>
+          <tbody>
+            <% @users.each do |user| %>
+              <tr>
+              <td><%= user.name %></td>
+              <td><%= user.email %></td>
+              <td><%= link_to 'Show', user %></td>
+              <td><%= link_to 'Destroy', user, method: :delete, data: { confirm: 'Are you sure?' } %></td>
```

Comment:
use double quote `" "` in Rails
remove whitespace

e sửa lại như này cho đúng convention và code nhìn đẹp hơn nhé:
```
<td>
  <%= link_to "Destroy", user, method: :delete, data: {confirm: "Are you sure?"} %>
</td>